src/main/java/com/appland/appmap/output/v1/Parameters.java

```diff
@@ -57,7 +57,8 @@ public Parameters(CtBehavior behavior) {
     // information. However, as of 20200822, other errors come up if
     // do hook them, so bail out here.
     if (locals == null) {
-      throw new NoSourceAvailableException("No local variables for " + fqn);
+        return;
```

Comment:
Due to this change, intrumented petclinic server is unable to run with java 13. hence smoke tests are failing.

Reply:
This causes the build to fail for both Java 8 and Java 1, as well.

The two methods you're hooking both take arguments. What is it that fails if this change isn't made?

Reply:
The behaviour passed to [Parameters](https://github.com/ta-Bruce/appmap-java/blob/b7cb5577ebe01c9a2373647e062d88b6b28a7e24/src/main/java/com/appland/appmap/output/v1/Parameters.java#L42) is actually refers to the target behaviour ` java.net.HttpUrlConnection#connect()` method , which has no arguments , and the two methods that we have added in appmap-java in [HttpClientRequest](https://github.com/ta-Bruce/appmap-java/blob/b7cb5577ebe01c9a2373647e062d88b6b28a7e24/src/main/java/com/appland/appmap/process/hooks/HttpClientRequest.java#L13) don't get processed by the code we are referring to, as per my knowledge.

Reply:
reverted back this change